case-lib/lib.sh

```diff
@@ -831,7 +831,7 @@ is_ipc4()
     fi
 
     dlogw "using pre kernel 6.9 backwards-compatible fallback with ipc_type kernel parameter"
-    if ipc_type=$(cat /sys/module/snd_sof_pci/parameters/ipc_type); then
+    if ipc_type=$(sudo cat /sys/module/snd_sof_pci/parameters/ipc_type); then
```

Comment:
This does not seem required. When running sof_insert/sof_remove I noticed `sudo` can be pretty slow, even slower on Fedora I think. Of course we should ideally cache the result of `is_ipc4()` but right now we don't.